src/py/pyodide/webloop.py

```diff
@@ -405,4 +405,19 @@ def set_event_loop(self, loop: Any) -> None:
         self._default_loop = loop
 
 
+def _initialize():
```

Comment:
```suggestion
def _initialize_event_loop():
```
to be more specific.